src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -6106,6 +6106,22 @@ export class ZoneServer2016 extends EventEmitter {
     }, 600000);
   }
 
+  useAmmoBox(client: Client, item: BaseItem) {
+    if (!this.removeInventoryItem(client.character, item)) return;
+
+    // Configure amount of ammo? Should be 60 by default.
```

Comment:
Yep should be in the config file

Reply:
Ammo boxes weren't used in this version and were leftover from 2015 / maybe early 2016, so if these are used it'll proabably only be on modded servers so idk if we should put it in config or not. Pistol bullets have the same model as the ammo box so it'd be confusing to spawn them with eachother.